sdk/src/main/java/io/dapr/actors/DaprAsyncClient.java

```diff
@@ -10,7 +10,7 @@
 /**
  * Interface for interacting with Dapr runtime.
  */
-interface DaprAsyncClient {
+public interface DaprAsyncClient {

```

Comment:
Let's not make these public, it's made public because of adding a runtime package,  if needed put classes in same package OR better if this can be split into calls from proxy to Dapr and from runtime to Dapr. Then they can stay in their individual packages.

Reply:
Split into 2 clients: app -> dapr and proxy -> dapr -> app.  The former is in the runtime dir and the other is in the dir above it.